03-lists.md

```diff
@@ -167,15 +167,49 @@ odds: [1, 3, 5, 7]
 
 This is different from how variables worked in lesson 1, and more similar to how a spreadsheet works.
 
-## Turn a string into a list {.challenge}
+> ## Turn a string into a list {.challenge}
 
-Use a for-loop to convert the string "hello" into a list of letters:
+> Use a for-loop to convert the string "hello" into a list of letters:
 
-~~~ {.python}
-["h", "e", "l", "l", "o"]
-~~~
-Hint: You can create an empty list like this:
+> ~~~ {.python}
+> ["h", "e", "l", "l", "o"]
+> ~~~
+> Hint: You can create an empty list like this:
+
+> ~~~ {.python}
+> my_list = []
+> ~~~
+
+Subsets of lists and strings can be accessed by specifying ranges of values in brackets, similar to how we accessed ranges of positions in a Numpy matrix. This is commonly referred to as "slicing" the list/string.
```

Comment:
Currently, our convention is to have all of the lesson material first, then all of the challenges at the end. Could you move this new material up so that it appears before the challenges?

Also (this is much more nit-picky), could you add some line breaks here? We currently don't do this very consistently, but in most of our `.md` files, we have newlines either after a certain number of characters, or when it makes sense for the sentence (see [this blog post](http://rhodesmill.org/brandon/2012/one-sentence-per-line/) for details). The main reason for this is so that if someone makes a change to this paragraph later on, it will be more obvious what has changed because diffs are done on a line-by-line basis.